docs/src/pages/[platform]/getting-started/installation/dependencies.android.mdx

```diff
@@ -20,6 +22,9 @@ Amplify UI Android requires Android API level 24 (Android 7.0) or above. The Amp
       dependencies {
           // Material3 dependency for theming the connected components
           implementation 'androidx.compose.material3:material3:1.0.1'
+
+          // Support for Java 8 features
```

Comment:
Do we need to callout who this is optional for, so they can leave it off if they don't need it?